src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -0,0 +1,344 @@
+import classnames from 'classnames';
+import { useCallback, useMemo, useState } from 'preact/hooks';
+
+import { ArrowLeftIcon, ArrowRightIcon } from '../../../../components/icons';
+import { IconButton, InputGroup } from '../../../../components/input';
+import Select from '../../../../components/input/SelectNext';
+import Library from '../../Library';
+
+const items = [
+  { id: '1', name: 'All students' },
+  { id: '2', name: 'Albert Banana' },
+  { id: '3', name: 'Bernard California' },
+  { id: '4', name: 'Cecelia Davenport' },
+  { id: '5', name: 'Doris Evanescence' },
+];
+
+function Select_({
+  disabled,
+  textOnly,
+  theItems = items,
```

Comment:
I think this might work without the `theItems` name here, since the left side is a property name and the right side is a different context. In any case, I would probably prefer to give the global variable the more distinct name than the local variable to avoid a conflict.

Reply:
Sounds reasonable. I can probably call the global one `defaultItems` or something like that.